index.bs

```diff
@@ -616,10 +630,40 @@ algorithm:
 
 ## Security Model Opt-Outs ## {#security-model-opt-outs}
 
-Issue: TODO: Fill in sub sections for the various security model opt-outs.
-This includes document.cookie access, how event.origin is serialized in
-postMessage, localStorage sharing, how to use old-school Origin header and ACAO
-header.
+For backwards compatibility, Suborigins provide several opt-opts from the
+standard security model. A developer can choose to use these opt-outs by
+specifying a <a>suborigin policy</a> in <a href="#the-suborigin-header">the
+suborigin header</a>
+
+Since these opt-outs weaken the security model of suborigins, developers SHOULD
+NOT use these options unless they are required to make their application work.
+
+The values of <a link-type="grammar">suborigin-policy-option</a> that may be
+present in a <a>suborigin policy</a> have the following effects:
+
+* '<dfn>`unsafe-postmessage-send`</dfn>' When a message is sent <i>from</i> a suborigin
+  namespace with <a>`unsafe-postmessage-send`</a> set, the `event.origin` value
+  of the receiver should be set to the serialized physical origin, not the
+  serialized suborigin value. However, the `event.suborigin` field should still
+  be set to the name of the suborigin namespace.
+
+* '<dfn>`unsafe-postmessage-receive`</dfn>' When a message is sent <i>to</i> a
+  frame with a `postMessage` `target` of a serialized physical origin, but not a
+  serialized suborigin, if the frame has an execution context with a suborigin
+  where the scheme, host, and port match the `target`, but it also has a
+  suborigin namespace, if <a>`unsafe-postmessage-receive`</a> is set, it will
+  still receive the message.
+
+* '<dfn>`unsafe-cookies`</dfn>' When an execution context with a suborigin
+  namespace has <a>`unsafe-cookies`</a> set, the
+  execution context should not have a fresh cookie jar for the suborigin
+  namespace, and instead, the cookie jar should be shared with the null
+  suborigin for the execution context.
```

Comment:
It would be nice to explicitly flesh out the set-cookie vs document.cookie use cases. Maybe leave a TODO?